cli/ioctl/cmd/block/height.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package block
+
+import (
+	"context"
```

Comment:
"context"
	"fmt"

	"github.com/spf13/cobra"
	"go.uber.org/zap"
	grpc "google.golang.org/grpc"

 	"github.com/iotexproject/iotex-core/pkg/log"
	pb "github.com/iotexproject/iotex-core/protogen/iotexapi"


This is our convention